### ISSUE TITLE
fail2ban service is broken (systemd backend fails)

### ISSUE THREAD

**Issue by anderspapitto** [NixOS/nixpkgs#11964]
fail2ban seems to be entirely non-functional, with the new systemd version. The error is

```
Dec 26 10:46:37 duncan fail2ban.jail[1058]: ERROR Backend 'systemd' failed to initialize due to No module named systemd
Dec 26 10:46:37 duncan fail2ban.jail[1058]: ERROR Failed to initialize any backend for Jail 'ssh-iptables'
```

I just upgraded to the latest nixos-unstable, and my fail2ban configuration reads

```
    fail2ban = {
      enable = true;
      jails.ssh-iptables = "enabled = true";
    };
```

**Comment by womfoo** [comment]
This has been addressed in 38a6b5f but has not made its way to the nixos-unstable channel yet. You should be set once its there.

**Comment by anderspapitto** [comment]
@womfoo thanks. For now I cherry-picked onto my local branch
